src/user/managing-dashboards.md

```diff
@@ -296,84 +259,64 @@ To view the chart, table or map in fullscreen, click on the **View fullscreen**
 
 ![](resources/images/dashboard/dashboard-item-menu-interpretations.png)
 
-> **Note**
-> 
->  This option may be disabled on your system if the system setting _Allow users to show dashboard favorite interpretations and details_ is unchecked.)
 
-The item will be expanded vertically underneath to show the description,
-interpretations and replies. You can like an interpretation, reply to an
-interpretation, and add your own interpretation. You can edit, share or delete
-your own interpretations and replies, and if you have moderator access,
-you can delete others’ interpretations.
+The item will be expanded vertically underneath to show the description, interpretations and replies. You can like an interpretation, reply to an interpretation, and add your own interpretation. You can edit, share or delete your own interpretations and replies, and if you have moderator access, you can delete others’ interpretations.
 
-It is possible to format the description field, and interpretations
-with **bold**, _italic_ by using the Markdown style markers \* and \_
-for **bold** and _italic_ respectively. The text field for writing new
-interpretations has a toolbar for adding rich text. Keyboard shortcuts
-are also available: Ctrl/Cmd + B and Ctrl/Cmd + I. A limited set of
-smilies is supported and can be used by typing one of the following
-character combinations: :) :-) :( :-( :+1 :-1. URLs are automatically
-detected and converted into a clickable link.
+It is possible to format the description field, and interpretations with **bold**, _italic_ by using the Markdown style markers \* and \_ for **bold** and _italic_ respectively. The text field for writing new interpretations has a toolbar for adding rich text. Keyboard shortcuts are also available: Ctrl/Cmd + B and Ctrl/Cmd + I. A limited set of smilies is supported and can be used by typing one of the following character combinations: :) :-) :( :-( :+1 :-1. URLs are automatically detected and converted into a clickable link.
 
-Interpretations are sorted in descending order by date, with the most recent shown on top.
-Interpretation replies are sorted in ascending order by date, with the oldest shown on top.
+Interpretations are sorted in descending order by date, with the most recent shown on top. Interpretation replies are sorted in ascending order by date, with the oldest shown on top.
 
 ![](resources/images/dashboard/dashboard-interpretations.png)
 
 ## Sharing a dashboard { #dashboard_sharing } 
 
-In order to share a dashboard with user groups, click on the **SHARE**
-button to the right of the dashboard title to display the dashboard
-sharing settings options. To share the dashboard with specific users or
-user groups, type in the name in the input field to add them to the
-dashboard sharing settings
+In order to share a dashboard with users and user groups, click on the **Share** button to the right of the dashboard title to display the *Sharing & Access* dialog.
```

Comment:
why does the modal say `Sharing & Access` with a capital A? 🤔

Reply:
Will be updated in the ui component